src/dd-win-prof/StackSamplerLoop.cpp

```diff
@@ -139,6 +139,11 @@ void StackSamplerLoop::CpuProfilingIteration()
 
             if (isRunning)
             {
+                // if it was waiting, reset the waiting timestamp
```

Comment:
minor comment is
if it was not waiting, reset the waiting timestamp to 0 (null value).
Though should we not consider the thread state ? Is unscheduled also waiting ?